administrator-guides/livechat/livechat-queues/README.md

```diff
@@ -1,9 +1,10 @@
 # Livechat Queue Types
 
-There two types of queue types on Rocket.Chat:
+There are three types of queue types on Rocket.Chat:
 
 - [Round robin [default]](#round-robin)
 - [Guest Pool (coming soon)](#guest-pool)
```

Comment:
I guess the `(coming soon)` from `Guest Pool` can also be removed.

Reply:
Removed.